gradle/libs.versions.toml

```diff
@@ -2,7 +2,6 @@
 agp = "8.3.0"
 kotlin = "1.8.10"
 coreKtx = "1.12.0"
-gms = "4.4.2"
```

Comment:
Watch out! You've removed GoogleMobileServices reference version, otherwise looks good to me!